tests/test_costs.py

```diff
@@ -149,3 +150,26 @@ def test_costs_5D_noisy(signal_bkps_5D_noisy, cost_name):
 def test_factory_exception():
     with pytest.raises(ValueError):
         cost_factory("bkd;s")
+
+
+# Test CostLinear
+def test_costlinear(signal_bkps_5D_noisy, signal_bkps_1D_noisy):
+    # Creation of data. For convinience, we use
+    # already generated signal_bkps_5D and signal_bkps_1D
+    signal_regressors, _ = signal_bkps_5D_noisy  # regressors
+    signal, bkps = signal_bkps_1D_noisy  # observed signal
+    n = signal.shape[0]  # number of samples
+
+    # Add intercept to regressors
+    X = np.append(signal_regressors, np.ones(n).reshape(-1, 1), axis=1)
+    # Stack observed signal and regressors.
+    # First dimension is the observed signal.
+    s = np.column_stack((signal.reshape(-1, 1), X))
```

Comment:
Not necessary to change it here but there exists a [simpler numpy shorthand](https://numpy.org/doc/stable/reference/generated/numpy.c_.html#numpy.c_) to concatenate arrays:

```python
s = np.c_[signal, signal_regressors, np.ones(n)]
```

Reply:
Will change it ! Was actually looking for an alternative to `np.append` !